src/utils/get-organized-arguments.ts

```diff
@@ -160,13 +160,17 @@ export default async function getOrganizedArguments(): Promise<OrganizedArgument
 						// Store details
 						previousOption = argument;
 						nextIsOptionValue = true;
+						nextValueType = details.type || undefined;
+						organizedArguments.options.push(option);
 
 						const arrayOrPromise =
 							typeof details.accepts === 'function' ? details.accepts() : details.accepts || undefined;
 						nextValueAccepts = arrayOrPromise instanceof Promise ? await arrayOrPromise : arrayOrPromise;
 
-						nextValueType = details.type || undefined;
-						organizedArguments.options.push(option);
+						// Error if accepts is not an array or absent and required
+						if ((details?.acceptsMultiple || nextValueAccepts) && !(nextValueAccepts instanceof Array)) {
+							throw new Error(`option['${option}'].accepts must resolve to an Array`);
```

Comment:
I don't think "array" is a proper noun here:

```suggestion
							throw new Error(`option['${option}'].accepts must resolve to an array`);
```